package.json

```diff
@@ -24,7 +24,7 @@
   "license": "MIT",
   "dependencies": {
     "loopback-connector": "^2.2.0",
-    "mongodb": "^2.0.35",
+    "mongodb": "2.1.21",
```

Comment:
Please use `~2.1.21`. If the there are any bugfixes publised in `2.1.x` series, we want to get them automatically. Unless there is a reason for pinning down the version to 2.1.21 only?